### DISCUSSION TITLE
[30 Giugno 2023] Cache

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/architettura-degli-elaboratori#63] [Cache (M-Z)]
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/0eab71ae-d2ac-4932-8cff-e3c2e69e7003)

**Comment by AglaiaNorza** [comment]
<img width="788" alt="C220628B-C621-4C85-A193-02B7A3DAAEF8" src="https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/75022526/8e88f146-9aa3-418e-ba81-71ffcbf0aebf">

**Comment by rimaout** [comment]
<img width="773" alt="image" src="https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/116072651/3ae7944d-fbd3-4022-bb49-890eb6791bdc">
